report/assets/styles.css

```diff
@@ -1689,23 +1633,53 @@
  transition: transform 300ms, stroke 300ms;
 }
 
-.lh-audit-group > summary > .lh-audit-group__summary > .lh-chevron .lh-chevron__line-right,
-.lh-audit-group[open] > summary > .lh-audit-group__summary > .lh-chevron .lh-chevron__line-left,
-.lh-audit > .lh-expandable-details .lh-chevron__line-right,
-.lh-audit > .lh-expandable-details[open] .lh-chevron__line-left {
+.lh-expandable-details .lh-chevron__line-right,
+.lh-expandable-details[open] .lh-chevron__line-left {
  transform: rotate(var(--chevron-angle-right));
 }
 
-.lh-audit-group[open] > summary > .lh-audit-group__summary > .lh-chevron .lh-chevron__line-right,
-.lh-audit > .lh-expandable-details[open] .lh-chevron__line-right {
+.lh-expandable-details[open] .lh-chevron__line-right {
   transform: rotate(var(--chevron-angle));
 }
 
-.lh-audit-group[open] > summary > .lh-audit-group__summary > .lh-chevron .lh-chevron__lines,
-.lh-audit > .lh-expandable-details[open] .lh-chevron__lines {
+
+.lh-expandable-details[open]  .lh-chevron__lines {
  transform: translateY(calc(var(--chevron-size) * -1));
 }
 
+.lh-expandable-details[open] {
+  animation: 300ms openDetails forwards;
+  padding-bottom: var(--default-padding);
+}
+
+@keyframes openDetails {
+  from {
+    outline: 1px solid var(--report-background-color);
+  }
+  to {
+   outline: 1px solid;
+   box-shadow: 0 2px 4px rgba(0, 0, 0, .24);
+  }
+}
+
+@media screen and (max-width: 780px) {
+  /* no black outline if we're not confident the entire table can be displayed within bounds */
+  .lh-expandable-details[open] {
+    animation: none;
+  }
+}
+
+.lh-expandable-details[open] summary, details.lh-clump > summary {
+  border-bottom: 1px solid var(--report-border-color-secondary);
+}
+details.lh-clump[open] > summary {
+  border-bottom-width: 0;
+}
+
+
+
+details .lh-clump-toggle::before {content: 'Show';}
+details[open] .lh-clump-toggle::before {content: 'Hide';}
```

Comment:
Some more strings that could be localized here

Reply:
pulled into separate PR https://github.com/GoogleChrome/lighthouse/pull/13308